### ISSUE TITLE
Kinship matrix in pcangsd

### ISSUE THREAD

**Issue by ThibauldMichel** [Rosemeis/pcangsd#72]
I am using PCAngsd v.0.95, and try to output a kinship matrix.

The kinship matrix command exist in PCAngsdv2 (-kinship):
http://www.popgen.dk/software/index.php/PCAngsdv2

But I cannot find the option in PCAngsd:
http://www.popgen.dk/software/index.php/PCAngsd

I guess the kinship matrix is given in a binary numpy format (file.npy), but the documentation is unclear on this subject.

**Comment by Rosemeis** [comment]
Hi,

The kinship estimation was unfortunately a prototype function that was never fully tested and has therefore been removed in the later versions. Sorry for the confusion!

Best,
Jonas

**Comment by jesserlasky** [comment]
We have already used the code for kinship matrix estimation from pcangsd v0.99
Can you provide a citation for the method used please? That way we can describe what we have done.

**Comment by Rosemeis** [comment]
Yes, it was an implementation of the PC-Relate estimator from [here](https://doi.org/10.1016/j.ajhg.2015.11.022) using our genotype posteriors.
